src/main/resources/log4j.properties

```diff
@@ -0,0 +1,10 @@
+# Root logger option
+log4j.rootLogger=DEBUG, stdout
```

Comment:
let's add saving logs to file as well
create the `logs` folder with the .log file inside
don't forget to add this folder to gitignore so that it is not pushed

Reply:
done